device-types/Dell/PowerEdge_C6420.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Dell
+model: PowerEdge C6420
+slug: dell_poweredge_c6420
+u_height: 2
+is_full_depth: false
+comments: |
+  [Dell C6420 Spec Sheet](https://www.delltechnologies.com/asset/en-us/products/servers/technical-support/dell-emc-poweredge-c6520-spec-sheet.pdf)
+console-server-ports:
+  - name: Rear USB
+    type: usb-a
```

Comment:
We are no longer modelling USB ports that are not console ports as console or console server ports